src/containers/NavContainer.js

```diff
@@ -29,6 +29,16 @@ const NavContainer = props => (
                   <Link to="/settings">Account</Link>
                 </li>
             }
+            <li className="nav-item">
+              <a
+                className="nav-link"
+                href="//faq.charlottejuniordevs.com"
```

Comment:
Unfortunately, I just read that GitHub doesn't support https for custom domains. 😢   Do you mind changing the link to go to `charlottejuniordevs.github.io/faq`?  That way the user can still have SSL.